presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/ThriftErrorCode.java

```diff
@@ -23,7 +23,8 @@
         implements ErrorCodeSupplier
 {
     THRIFT_SERVICE_CONNECTION_ERROR(1, EXTERNAL),
-    THRIFT_SERVICE_INVALID_RESPONSE(2, EXTERNAL);
+    THRIFT_SERVICE_INVALID_RESPONSE(2, EXTERNAL),
+    THRIFT_GENERIC_REMOTE_SERVER_ERROR(3, EXTERNAL);
```

Comment:
How about naming this `THRIFT_SERVICE_GENERIC_ERROR` to match the others?

Reply:
How about `THRIFT_SERVICE_GENERIC_REMOTE_ERROR`? Basically we want to emphasize that it's an error in the remote thrift server side.

Reply:
Sounds good